Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberViewModel.swift

```diff
@@ -9,7 +9,7 @@
 import Checkout
 
 protocol CardNumberViewModelDelegate: AnyObject {
-  func update(cardNumber: String, scheme: Card.Scheme)
+  func update(cardNumber: String?, scheme: Card.Scheme)
```

Comment:
Could we pass an empty String for cardNumber if we don't have one? Why does it need to be nil?

Reply:
if we need to update payment screen with invalid value for cardNumber to update pay button

Reply:
Ok, what is *invalid value*? 

In my understanding invalid value is an input that doesn't quite make it as a card number. 
Nil is not an input that doesn't make it as a card number, nil is the absence of such an input.